Assets/World/Buildings/Lumberjack/Forester/ForesterMovmentFrames.tres

```diff
@@ -0,0 +1,341 @@
+[gd_resource type="SpriteFrames" load_steps=65 format=3 uid="uid://b7inbywvvhts2"]
+
+[ext_resource type="Texture2D" uid="uid://7hnp6hwaxcsv" path="res://Assets/World/Buildings/Lumberjack/Sprites/Move/0/0001.png" id="1_xxwri"]
```

Comment:
combine all animation frames to a single sprite

Reply:
done